src/sass/public/UI/header/_sidebar.scss

```
@@ -0,0 +1,61 @@
+@use 'base/variables' as v;
+@use 'base/mixins' as m;
+
+.sidebar {
+    opacity: 0;
+    height: 0;
+    transition: opacity 500ms ease-in-out;
+    overflow: hidden;
+
+    &.show {
+        padding: 10rem 2rem 2rem 2rem;
+        opacity: 1;
+        height: 100%;
+        width: 100%;
+        position: fixed;
+        top: 0;
+        left: 0;
+        background-color: rgb(0 0 0 / 0.90);
```

```suggestion
        padding: 2rem; // Dont use excesive padding
        opacity: 1;
        height: 100%;
        width: 100%;
        position: fixed;
        top: 60px; // create a sass or css variable to save the header height
```
dont use excesive padding use the header size instead. Create a sass or css variable to save the header height for mobile and desktop